pjsip/include/pjsip-ua/sip_siprec.h

```diff
@@ -0,0 +1,135 @@
+ * 
+ * @return               PJ_TRUE if a label exists in the SDP.
+ */
+PJ_DECL(pj_status_t) pjsip_siprec_verify_sdp_attr_label(pjmedia_sdp_session *sdp);
+
+
+/**
+ * Verifies that the incoming request has the siprec value in the Require header
+ * and "+sip.src" parameter exist in the Contact header.
+ * If both conditions are met, according to RFC 7866,
+ * the INVITE request is a siprec. Otherwise, no changes are made to the request.
+ * if INVITE request is a siprec must have media attribute label exist in the SDP
+ *
+ * @param rdata         The incoming request to be verified.
+ * @param metadata      The siprec metadata information
+ * @param sdp_offer     The SDP media.
+ * @param options       The options argument is bitmask combination of SIP 
+ *                      features in pjsip_inv_option enumeration
+ * @param dlg           The dialog instance.
+ * @param endpt         Media endpoint instance.
+ * @param p_tdata       Upon error, it will be filled with the final response
+ *                      to be sent to the request sender.
+ * 
+ * @return              The function returns the following:
+ *                      - If the request includes the value siprec in the Require header
+ *                        and also includes "+sip.src" in the Contact header.
+ *                        PJ_SUCCESS and set PJSIP_INV_REQUIRE_SIPREC to options
+ *                      - Upon error condition (as described by RFC 7866), the
+ *                        function returns non-PJ_SUCCESS, and \a p_tdata 
+ *                        parameter SHOULD be set with a final response message
+ *                        to be sent to the sender of the request.
+ */
+PJ_DECL(pj_status_t) pjsip_siprec_verify_request(pjsip_rx_data *rdata,
+                                                pj_str_t *metadata,    
+                                                pjmedia_sdp_session *sdp_offer,                                       
+                                                unsigned *options,
+                                                pjsip_dialog *dlg,
+                                                pjsip_endpoint *endpt,
+                                                pjsip_tx_data **p_tdata);
+
+
+/**
+ * Find siprec metadata information from the message body
+ * with "rs-metadata+xml" Content-Type.
+ *
+ * @param pool               Pool to allocate memory.
+ * @param body               The message body.
+ * @param metadata           The siprec metadata
+ */
+PJ_DECL(void) pjsip_siprec_find_metadata(pj_pool_t *pool,
+                                        pjsip_msg_body *body,
+                                        pj_str_t* metadata);
+
+
+/*
+ * Counts the number of audio and video streams in the SDP
+ * 
+ * @param sdp         The SDP media.
+ * @param maudcnt     media audio count.
+ * @param mvidcnt     media video count
+ */
+PJ_DECL(void) pjsip_siprec_count_media(pjmedia_sdp_session *sdp,                                     
```

Comment:
Remove this since the API is not used anywhere, and also shouldn't be the responsibility of siprec.

Reply:
remove it